src/DateParser.jl

```diff
@@ -1,6 +1,6 @@
 module DateParser
 
-using Base.Dates
+using Dates
```

Comment:
We should do `using Compat.Dates` so we can have one revision which is 0.6 compatible and up. You'll also need to change the REQUIRE file to reflect this.